app/attendees/[id]/page.tsx

```
@@ -127,6 +146,14 @@ export default function AttendeePage() {
               <AttendeeForm attendee={attendee} onSuccess={handleEditSuccess} />
             </TabsContent>
           </Tabs>
+          {/* botones confirmacion */}
+          <div className="grid grid-cols-1 gap-4 lg:grid-cols-2">
+            <ConfirmationButton userId={id} />
+            <Button variant={"default"} className="w-full flex gap-2">
+              <Gift />
+              <>Entregar regalo</>
+            </Button>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)